resources/authgear/templates/en/web/authflowv2/enter_password.html

```diff
@@ -6,5 +6,32 @@ <h1 class="screen-title">Enter Password</h1>
 <div class="w-[400px] p-6 space-y-2">
   <button class="primary-btn w-full">Primary</button>
   <button class="primary-btn w-full" disabled>Primary Disabled</button>
+  <button class="secondary-btn w-full">
+    <svg
```

Comment:
Can we put this svg into a css class? (Maybe check how tabler write the css?)

Reply:
We used tabler icon font. So we just use `color` to change the tint color.

Reply:
Opened another issue to implement the icons.
https://github.com/authgear/authgear-server/issues/3568